broker/config/settings.yml

```diff
@@ -187,6 +187,11 @@ defaults: &defaults
       key: <%= certificate('broker.key') %>
       cert: <%= certificate('broker.crt') %>
 
+  apiserver:
+    ip: '127.0.0.1'
+    port: 9443
+    encryption:
+      password: 'ABCDEFGH'
```

Comment:
This encryption key must remain constant once generated, right? So this needs to be marked as immutable. Is it possible to somehow make the encryption not dependent on an immutable constant?

Reply:
good point...actually I was also thinking of the same. the problem with config is that one can regenerate and change it and then we may loose old bindings!

I can't think of any option as such. Do you have anything in mind? @jagadish-kb @kamath-prasad  may chime in here.

Reply:
@subhankarc : Not sure how exactly this methods will be used in the bind operation . One option I could think of is - can we store the keys/vector along with the encrypted string ? Maybe as part of bind credentials ? Then we can generate these keys on the fly ?

Reply:
If the key is at the same place, it would be very easy to attack right?